From 82dd350d4502329237c3918592a5b7354e936e7c Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sat, 20 Jan 2018 09:06:21 -0500
Subject: [PATCH] Fix detection of Python 2

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index daa3143..d2e8b70 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ if (NOT PYTHON_DESIRED)
 endif()
 
 if (${PYTHON_DESIRED} STREQUAL "2")
-	FIND_PACKAGE (PythonInterp REQUIRED)
+	FIND_PACKAGE (PythonInterp 2.7 REQUIRED)
 else()
 	SET(Python_ADDITIONAL_VERSIONS 3.3)
 	FIND_PACKAGE(PythonLibs 3.0)
-- 
2.12.3

